### ISSUE TITLE
App Crashes At Launch

### ISSUE THREAD

**Issue by misteam** [jenius-apps/nightingale-rest-api-client#194]
Windows 10 Pro version 21H1 build 19043.1023

The app crashes right at splash screen when launching. It had worked previously, but stopped a few weeks ago. The build mentioned above is an update that occurred after the crash at launch issue started to occur, so the issue spans updates of my os.

**Comment by dpaulino** [comment]
Interesting. Can you try installing an older version of nightingale to see if the crash is still happening or not? You can go to the GitHub releases to install an older version.

**Comment by misteam** [comment]
I'm sorry but I'm not finding releases anywhere? I'm not too familiar with gitHub. Thank you for the quick response.

**Comment by dpaulino** [comment]
https://github.com/jenius-apps/nightingale-rest-api-client/releases

You'll need to uninstall your existing version first before installing an older build.

**Comment by misteam** [comment]
Thank you so much for getting back to me. I installed the prior version 4.3.7 and the app is functional again.

**Comment by dpaulino** [comment]
very strange. What happens if you uninstall the app and then re-install the latest version? does the latest still crash? I'm wondering if the uninstallation fixed the issue

**Comment by misteam** [comment]
No, uninstalling the 4.3.7 version, and re-installing the Microsoft Store version still results in an app crash for me.
